### PR TITLE
Thinking about replacements

### PR DESCRIPTION
**Pull request by Galithil**
This is the basic framework required to replace FSUL, the script that updates the flowcell db with the LIMS data.

### CODE REVIEW

File: scripts/flowcell_sql_upload.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+"""Script replacing flowcell_summary_uppload_LIMS.py
```
Comment (ewels) [inline scripts/flowcell_sql_upload.py]: Typo `uppload` -> `upload` :wink:

**Comment by ewels** [comment]
Looks nice! A few more comments explaining what the different functions are doing might be nice, as it was a bit tough to work out what was going on in each chunk. But can merge if you want :+1:

File: scripts/flowcell_sql_upload.py (hunk 1 above)
Comment (Galithil) [inline scripts/flowcell_sql_upload.py]: Actually no, that's how the script is named...

Comment (ewels) [inline scripts/flowcell_sql_upload.py]: :disappointed:

**Pull request merged**